services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -49,7 +49,7 @@ func DefaultConfig() *config.Config {
 				},
 				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
```

Comment:
Could you please remove the `preview` app from the list of apps and add it to the external apps struct as described here:
https://owncloud.dev/clients/web/deployments/oc10-app/#preview-app
(works the same way for ocis as for oc10, was just needed for oc10 first so I added it there back then)

Reply:
I changed the config. Is that how you wanted it?

Reply:
still needs some changes, see comments 😅